dts/bindings/adc/lltc,ltc2451.yaml

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2023 Brill Power Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Linear Technology LTC2451 ADC
+
+compatible: "lltc,ltc2451"
+
+include: base.yaml
```

```suggestion
include: i2c-device.yaml
```

and remove reg property.

---

Has been changed!